services/utils/twemoji.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * modified version of https://unpkg.com/twemoji@13.1.0/dist/twemoji.esm.js.
+ */
+
+/* ! Copyright Twitter Inc. and other contributors. Licensed under MIT */
+
+// this file added in: https://github.com/EddieHubCommunity/BioDrop/pull/9500
+
+const U200D = String.fromCharCode(8205);
+const UFE0Fg = /\uFE0F/g;
+
+export function getIconCode (char) {
+  return toCodePoint(!char.includes(U200D) ? char.replace(UFE0Fg, "") : char);
+}
+
+function toCodePoint (unicodeSurrogates) {
+  const r = [];
+  let c = 0;
+  let i = 0;
+  let p = 0;
+
+  // eslint-disable-next-line no-loops/no-loops
+  while (i < unicodeSurrogates.length) {
+    c = unicodeSurrogates.charCodeAt(i++);
+    if (p) {
+      r.push((65536 + ((p - 55296) << 10) + (c - 56320)).toString(16));
+      p = 0;
+    } else if (55296 <= c && c <= 56319) {
+      p = c;
+    } else {
+      r.push(c.toString(16));
+    }
+  }
+  return r.join("-");
+}
+
+export const apis = {
+  twemoji: (code) =>
+    `https://cdnjs.cloudflare.com/ajax/libs/twemoji/14.0.2/svg/${
+      code.toLowerCase()
+    }.svg`,
+  openmoji: "https://cdn.jsdelivr.net/npm/@svgmoji/openmoji@2.0.0/svg/",
+  blobmoji: "https://cdn.jsdelivr.net/npm/@svgmoji/blob@2.0.0/svg/",
+  noto: "https://cdn.jsdelivr.net/gh/svgmoji/svgmoji/packages/svgmoji__noto/svg/",
+  fluent: (code) =>
+    `https://cdn.jsdelivr.net/gh/shuding/fluentui-emoji-unicode/assets/${
+      code.toLowerCase()
+    }_color.svg`,
+  fluentFlat: (code) =>
+    `https://cdn.jsdelivr.net/gh/shuding/fluentui-emoji-unicode/assets/${
+      code.toLowerCase()
+    }_flat.svg`,
+};
```

Comment:
This API here is extra code, yet I kept them OpenSauced in the hope of using them later and just to keep the code somehow similar to Vercel, yet for biodrop if the mainaters (sara or eddie) like that we could have an emoji configuration in the settings of the persona profile, could be some cute feature to have.

Reply:
This be the case, then I guess we can leave the `twemoji` util content as is 😉